packages/api/src/controllers/playback.ts

```diff
@@ -316,6 +320,29 @@ app.get("/:id", async (req, res) => {
   const origin = req.headers["origin"] ?? "";
   const isEmbeddablePlayer = embeddablePlayerOrigin.test(origin);
 
+  if (withRecordings) {
```

Comment:
Could this be easily moved to inside the `getPlaybackInfo` function? There's probably a place in there where we handle streams explicitly, and I think we should avoid fetching the same data multiple times on a request